src/FunderMaps.WebApi/Controllers/Application/ReviewerController.cs

```diff
@@ -50,8 +50,9 @@ public ReviewerController(IMapper mapper, AuthManager authManager, OrganizationM
         public async Task<IActionResult> GetAllAsync([FromQuery] PaginationModel pagination)
         {
             // Assign.
+            var roles = new OrganizationRole[]{ OrganizationRole.Verifier, OrganizationRole.Superuser };
```

Comment:
Might it be a good idea to create a function for getting a role and all its super-roles? Hardcoding an array wherever we need it may not be the most bulletproof solution.

Reply:
There is no other business logic except for this controller